src/components/AppContent.js

```diff
@@ -1,13 +1,60 @@
 import '../css/App.css';
+import React, { useState } from 'react';
 import KevContainer from './KevContainer.js';
 import QuoteContainer from './QuoteContainer.js';
+import StartButton from './StartButton.js';
+import ItsKevButton from './ItsKevButton.js';
+import NotKevButton from './NotKevButton.js';
+import Points from './Points.js';
+import ResetButton from './ResetButton.js';
+import newQuote from '../utils/quoteGenerator';
+
+
 
 const AppContent = () => {
+
+  const placeholderQuote = {author: "Placeholder", quotation: "Hit the button to start."};
```

Comment:
Not necessarily a biggie, but constant values that never change (like this placeholderQuote) should usually be created outside of the component. 

Although minor, every time this component is rerendered this constant will be recreated, making this less optimised than it could be